net/proxy_resolution/proxy_config_service_tor.cc

```diff
@@ -140,10 +141,12 @@ std::string ProxyConfigServiceTor::CircuitIsolationKey(const GURL& url) {
   url::Origin url_origin = url::Origin::Create(url);
   net::NetworkIsolationKey network_isolation_key(url_origin, url_origin);
 
-  const base::Optional<url::Origin>& network_isolation_key_origin =
+  const base::Optional<net::SchemefulSite>& schemeful_site =
       network_isolation_key.GetTopFrameSite();
-  DCHECK(network_isolation_key_origin.has_value());
-  return network_isolation_key_origin->host();
+  DCHECK(schemeful_site.has_value());
+  std::string host = GURL(schemeful_site->Serialize()).host();
+  DCHECK(host == schemeful_site->GetInternalOriginForTesting().host());
```

Comment:
why are we calling `GetInternalOriginForTesting` here? I'm not sure we should be using this method in a non-test environment

Reply:
I think it can be removed. This is just used to double check `host` we use is equivalent to `site_as_origin_.host()`

Reply:
Removed in 0ec9fbad4fd9d6ef0f3130a83be0d422df61eb04